completion/available/kind.completion.bash

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+if command -v kind &>/dev/null
+then
```

Comment:
I think its more readable to put this in the same line as the `if`, i.e `if _command_exists kind; then`

Reply:
I totally agree. I just took the command from the other file in the repository.